T4MVCHostMvcApp/T4MVC Files/T4MVC.tt

```diff
@@ -1298,7 +1298,7 @@ PopIndent();
 
 if (!settings.ExcludedStaticFileExtensions.Any(extension => projectItem.Name.EndsWith(extension, StringComparison.OrdinalIgnoreCase))) {
 
-    var extension = projectItem.Name.Substring(projectItem.Name.LastIndexOf("."));
+    var extension = Path.GetExtension(projectItem.Name.LastIndexOf("."));
```

Comment:
YES! :-/ No Idea how I managed to commit that - thought I'd copied the whole line over from my working project - will resubmit ...